platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/consensus/ConsensusTests.java

```diff
@@ -44,16 +45,26 @@ class ConsensusTests extends PlatformTest {
      */
     private final int NUM_ITER = 1;
 
+    private AtomicBoolean ignoreNoSuperMajorityMarkerFile = new AtomicBoolean(false);
```

Comment:
I'm not grokking why these need to be atomic

If the atomic is necessary, though, the members may be `final`

Reply:
I've made  it a normal boolean since all methods are run in separate instances of the class.